core/lib/spree/money.rb

```diff
@@ -10,7 +10,16 @@ class <<self
       attr_accessor :default_formatting_rules
 
       def from_money(money)
-        new(money.to_d, currency: money.currency.iso_code)
+        new(money)
+      end
+
+      def parse(amount, currency = Spree::Config[:currency])
+        new(parse_to_money(amount, currency))
+      end
+
+      # @api private
```

Comment:
It's called both from an instance of Money, and from the class so it can't be :(

We can make it private once the deprecation is removed.